test/spell/DisableV2Eth.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.28;
+
+import {IShareToken} from "src/spoke/interfaces/IShareToken.sol";
+
+import {DisableV2Common} from "./DisableV2Common.sol";
+
+/// @notice Ethereum-specific spell that disables V2 permissions for both JTRSY_USDC and JAAA_USDC
+contract DisableV2Eth is DisableV2Common {
+    // JAAA configuration (only exists on Ethereum mainnet)
+    IShareToken public constant JAAA_SHARE_TOKEN = IShareToken(0x5a0F93D040De44e78F251b03c43be9CF317Dcf64);
+
+    address public constant JTRSY_VAULT_ADDRESS = address(0x1d01Ef1997d44206d839b78bA6813f60F1B3A970);
```

Comment:
TODO: Seems like this vault address is incorrect due to mismatch with spell of #540: https://github.com/centrifuge/protocol-v3/pull/540/files#r2230766243